app/main.py

```
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_needed = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Error: The `OutdatedVaccineError` exception is not handled explicitly. According to the requirements, all custom exceptions must be handled, and the return message for an outdated vaccine should be specific. Add an `except OutdatedVaccineError` block before the general `VaccineError` block with the appropriate return message.